### ISSUE TITLE
Add license to pom.xml

### ISSUE THREAD

**Issue by jjloneman** [HubSpot/prettier-maven-plugin#15]
## Problem
My enterprise's Maven repository manager automatically rejects requesting this package since it does not contain a license property in the pom, though one is provided in this Github repo.

## Solution
A license property should be added to `pom.xml` like so:

```xml
<project ...>
    ...
    <licenses>
        <license>
            <name>Apache License, Version 2.0</name>
            <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
        </license>
    </licenses>
    ...
</project>
```

**Comment by jhaber** [comment]
Thanks for reporting this issue, I just pushed 93bdb1d2cae6efe4a6a8b31a025ed2e2d2782749 Do you also need me to cut a new release with this change?

**Comment by jjloneman** [comment]
Yes, that would be great, thanks!

**Comment by jhaber** [comment]
I just released version 0.9 and it should show up in Maven Central in a few hours

**Comment by jjloneman** [comment]
Awesome, thank you and keep up the great work! :)
